packages/app/src/visualizer/utils.ts

```diff
@@ -121,21 +118,34 @@ function getSupportedNxVis(
   return NEXUS_VIS[hasMinDims(dataset, 2) ? imageVis : spectrumVis];
 }
 
-function getImplicitDefaultChild(children: Entity[]): Entity | undefined {
-  const groups = children.filter(isGroup);
+function getImplicitDefaultChild(
+  children: Entity[],
+  attrValueStore: AttrValuesStore
+): Entity | undefined {
+  const nxGroups = children
+    .filter(isGroup)
+    .filter((g) => hasAttribute(g, 'NX_class'));
```

Comment:
We're only fetching attribute values of groups that we know have an `NX_class` attribute